[2024-10-30T11:17:31.452+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-30T11:17:31.493+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: News-Sentiment-Pipeline.store_news_sentiment_data_s3 manual__2024-10-30T11:17:20.425384+00:00 [queued]>
[2024-10-30T11:17:31.502+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: News-Sentiment-Pipeline.store_news_sentiment_data_s3 manual__2024-10-30T11:17:20.425384+00:00 [queued]>
[2024-10-30T11:17:31.503+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-30T11:17:31.514+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): store_news_sentiment_data_s3> on 2024-10-30 11:17:20.425384+00:00
[2024-10-30T11:17:31.537+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=20888) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-30T11:17:31.535+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'News-Sentiment-Pipeline', 'store_news_sentiment_data_s3', 'manual__2024-10-30T11:17:20.425384+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/news_sentiment_pipeline.py', '--cfg-path', '/tmp/tmpv0bscprf']
[2024-10-30T11:17:31.540+0000] {standard_task_runner.py:91} INFO - Job 284: Subtask store_news_sentiment_data_s3
[2024-10-30T11:17:31.540+0000] {standard_task_runner.py:63} INFO - Started process 20900 to run task
[2024-10-30T11:17:31.569+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-30T11:17:31.632+0000] {task_command.py:426} INFO - Running <TaskInstance: News-Sentiment-Pipeline.store_news_sentiment_data_s3 manual__2024-10-30T11:17:20.425384+00:00 [running]> on host cf1c99cdc1a8
[2024-10-30T11:17:31.754+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='News-Sentiment-Pipeline' AIRFLOW_CTX_TASK_ID='store_news_sentiment_data_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-10-30T11:17:20.425384+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-30T11:17:20.425384+00:00'
[2024-10-30T11:17:31.756+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-30T11:17:31.948+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-10-30T11:17:34.735+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-30T11:17:34.738+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-30T11:17:34.766+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=News-Sentiment-Pipeline, task_id=store_news_sentiment_data_s3, run_id=manual__2024-10-30T11:17:20.425384+00:00, execution_date=20241030T111720, start_date=20241030T111731, end_date=20241030T111734
[2024-10-30T11:17:34.846+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-30T11:17:34.886+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-30T11:17:34.888+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
